LocalExtensions.php

```diff
@@ -194,6 +194,21 @@
 
 if ( $wmgUseCommentStreams ) {
 	wfLoadExtension ( 'CommentStreams' );
+	$wi->config->settings['wgManageWikiNamespacesAdditional']['default']['wgCommentStreamsAllowedNamespaces'] = [
+		'name' => 'Can CommentStreams in this namespace?',
```

Comment:
It should be "Can comments appear in this namespace?"

Reply:
Done